src/main/java/functionalities/SniffTasks.java

```diff
@@ -10,7 +12,12 @@ public class SniffTasks {
     public void addAppointment() {
     }
 
-    public void removeAppointment() {
+    public void removeAppointment(int apptNum) throws SniffException {
+        try {
+            appointments.remove(apptNum);
+        } catch (IndexOutOfBoundsException e) {
+            throw new SniffException(" The remove command description is invalid!");

```

Comment:
LGTM
I like how you added try catch statements for error handling while adding functionality for removing appointment!